app/components/exhibit_navbar_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<div id="site-navbar" class="site-navbar mt-auto py-2">
+    <% if helpers.should_render_spotlight_search_bar? %>
+      <div class="container d-flex">
+        <%= render search_component.new(
+          url: helpers.search_action_url,
+          advanced_search_url: helpers.search_action_url(action: 'advanced_search'),
+          params: helpers.search_state.params_for_search.except(:qt),
+          autocomplete_path: suggest_index_catalog_path
+        ) %>
+        <%#<%= render SearchTipsLinkComponent.new %>
```

Comment:
We want search tips to display. Looks like it was commented out maybe unintentionally.

Reply:
Yes. That was unintentional. It's been readded.